manifests/system-upgrade-controller.yaml

```diff
@@ -58,13 +58,21 @@ spec:
             nodeSelectorTerms:
               - matchExpressions:
                   - {key: "node-role.kubernetes.io/master", operator: In, values: ["true"]}
+              - matchExpressions:
+                  - {key: "node-role.kubernetes.io/controlplane", operator: In, values: ["true"]}
```

Comment:
Are you sure about the label? I'm using `k3s` and the label is `node-role.kubernetes.io/control-plane` (notice **-** between word)

Reply:
this is what I see, machines are running CentOS7 with docker and kubernetes installed from rancher.

```
Name:               *****
Roles:              controlplane,etcd
Labels:             beta.kubernetes.io/arch=amd64
                    beta.kubernetes.io/os=linux
                    kubernetes.io/arch=amd64
                    kubernetes.io/hostname=*****
                    kubernetes.io/os=linux
                    node-role.kubernetes.io/controlplane=true
                    node-role.kubernetes.io/etcd=true
Annotations:        node.alpha.kubernetes.io/ttl: 0
                    rke.cattle.io/external-ip: *****
                    rke.cattle.io/internal-ip: *****
                    volumes.kubernetes.io/controller-managed-attach-detach: true
CreationTimestamp:  Mon, 11 Jan 2021 15:05:13 -0600
Taints:             node-role.kubernetes.io/etcd=true:NoExecute
                    node-role.kubernetes.io/controlplane=true:NoSchedule
Unschedulable:      false
```

Reply:
Yeah that was a bug, should have been since corrected.